module/commentPopup.js

```diff
@@ -0,0 +1,98 @@
+                  <form id="comment-form">
+                  <label for="name">Name:</label>
+                  <input type="text" id="name" name="name">
+                  <label for="comment">Comment:</label>
+                  <textarea id="comment" name="comment"></textarea>
+                  <button type="submit">Submit</button>
+                  </form>
+                  `;
+        const commentForm = document.getElementById('comment-form');
+        commentForm.addEventListener('submit', async (event) => {
+          event.preventDefault();
+          const name = document.getElementById('name').value;
+          const comment = document.getElementById('comment').value;
+
+          const uniqueId = 'iyIoaiC1J6eUiPDhXO68';
+          const baseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi';
+          const response = await fetch(`${baseUrl}/apps/${uniqueId}/comments`, {
+            method: 'POST',
+            headers: {
+              'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({
+              item_id: id,
+              username: name,
+              comment,
+            }),
+          });
+          if (!response.ok) {
+            throw new Error(`Error posting comment: ${response.statusText}`);
+          }
+          const commentElement = CommentsPopup.createCommentElement(name, comment, 'A few moments ago');
+          const comments = document.querySelector('.comments');
+          comments.appendChild(commentElement);
+
+          document.getElementById('comment-form').reset();
+        });
+        const url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi';
+        const response = await fetch(`${url}/apps/iyIoaiC1J6eUiPDhXO68/comments?item_id=${id}`);
+        let comments = await response.json();
+        if (!comments.length) {
+          comments = [];
+        }
+        comments.forEach((comment) => {
+          const comments = document.querySelector('.comments');
+
+          const commentElement = CommentsPopup.createCommentElement(comment.username,
+            comment.comment, comment.creation_date);
+          comments.appendChild(commentElement);
+          const counter = document.querySelectorAll('.comment-counter');
+          document.querySelector('h4').textContent = `Comments (${counter.length})`;
+        });
+      });
+    });
+  }
+
+  static createCommentElement(username, comment, creationDate) {
+    const commentElement = document.createElement('div');
+    commentElement.className = 'comment-counter';
+    commentElement.innerHTML = `${creationDate} <strong>${username}</strong>: ${comment} `;
+    return commentElement;
+  }
+
+  static async movieId(id) {
+    try {
+      const request = await fetch(`https://api.tvmaze.com/shows/${id}`);
+      const moive = await request.json();
+      return moive;
+    } catch (error) {
+      return error;
+    }
+  }
+}
```

Comment:
- [ ] [OPTIONAL] Awsome project team! however, I will recommend that you increase the width of the modal, it looks rather small and the comments passed into it needs their font sizes increased. This will give it a better UI vie and give a better UX for the user.

<img width="1216" alt="Screenshot 2023-03-25 at 16 12 50" src="https://user-images.githubusercontent.com/58692590/227726661-3d03ab8e-6707-4479-a6a7-e996188d17ee.png">